### PR TITLE
Add missing debug modes

### PR DESCRIPTION
**Pull request by haslinghuis**
Add missing debug modes.

Fixes: #3010

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by Asizon — DISMISSED** [review]

**Review by McGiverGim — APPROVED** [review]

File: src/js/tabs/onboard_logging.js
Comment (McGiverGim) [inline src/js/tabs/onboard_logging.js:356]: Here we don't use the DEBUG word. It's a list of debug modes, so please remove it.
```suggestion
                {text: "GPS_RESCUE_VELOCITY"},
                {text: "GPS_RESCUE_HEADING"},
                {text: "GPS_RESCUE_TRACKING"},
                {text: "ATTITUDE"},
```

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/js/tabs/onboard_logging.js
Comment (haslinghuis) [reply src/js/tabs/onboard_logging.js]: Thanks. Updated.

**Review by McGiverGim — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
